redis/redis-master.yaml

```diff
@@ -1,24 +1,20 @@
-apiVersion: v1beta1
-id: redis-master-pod
-kind: Pod
-desiredState:
-  manifest:
-    version: v1beta1
-    id: redis-master-pod
-    containers:
-      - name: redis-master
-        image: redis
-        ports:
-          - containerPort: 6379
-        volumeMounts:
-            # name must match the volume name below
-          - name: redis-persistent-storage
-            # mount path within the container
-            mountPath: /data/redis
-    volumes:
-      - name: redis-persistent-storage
-        source:
-          emptyDir: {}
-labels:
+apiVersion: v1
```

Comment:
It doesn't need to, but this is what the gb app does, making the case that you get the auto-restarts this way (it's also made clear that this redis installation is not persistent).
I was thinking to show the same-- I think I'll leave it this way, but add more explication.